tests/otel_integrations/test_django.py

```diff
@@ -79,6 +81,36 @@ def test_good_route(client: Client, exporter: TestExporter, metrics_reader: InMe
                     'aggregation_temporality': 1,
                 },
             },
+            {
+                'name': 'http.server.request.duration',
+                'description': 'Duration of HTTP server requests.',
+                'unit': 's',
+                'data': {
+                    'data_points': [
+                        {
+                            'attributes': {
+                                'http.request.method': 'GET',
+                                'url.scheme': 'http',
+                                'network.protocol.version': '1.1',
+                                'http.route': 'django_test_app/<int:item_id>/',
```

Comment:
This is what this is all about. Note that it's a whole new metric name. The old duration metrics are still above, unchanged, without anything similar to `http.route`.